code/modules/economy/EFTPOS.dm

```diff
@@ -268,7 +268,5 @@
 				playsound(src, 'sound/machines/chime.ogg', 50, 1)
 				src.visible_message("\icon[src] \The [src] chimes.")
 				transaction_paid = 1
-	else
```

Comment:
calling parent in a trailing else seems valid, why remove this?

Reply:
this code was never reached.